conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
This may need to be referred to legal. If the code was written during work hours or on equipment provided by SUSE LLC, they will generally own copyright (and this is usually stipulated as part of employment agreements). It's not permissible to change the statement without approval from the copyright owner (being the legal representative acting on behalf of SUSE LLC). This is especially true if the code was extracted from runC and already carries this sort of statement. This is generally why many projects require a Contributor License Agreement or at least a Developer Certificate of Origin.

@cyphar I don't have any personal opinion on individual copyright statements in files, but I will say that some vendors (Debian in particular) require a comprehensive machine-readable copyright file, which includes the names of all rights holders, and having additional copyright statements means more work for packagers. Just something to consider.